package.json

```diff
@@ -25,7 +26,8 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "deploy": "gh-pages -d build"
+    "deploy": "gh-pages -d build",
+    "cypress:open": "cypress open"
```

Comment:
Back in November, I had a custom `./scripts/cypress` runner that would use the API to open the GUI when in development, or run headless in CI.

Time prevented me from implementing that here.